Assets/Scripts/GameElements/PlayerController.cs

```diff
@@ -160,6 +185,30 @@ private void CheckForWingInput()
         }
     }
 
+    private void CheckForPlayerMash()
+    {
+        float delta = (buttonPressedTime - lastButtonPressedTime);
+        if (delta == 0)
+        {
+            delta = 0.1f;
+        }
+
+        // TODO: one day make a better function for this jesus
+        int mashValue = (int) (1.0f / delta);
```

Comment:
I mentioned this in the meeting but if we want it to just be a "count of how many times you've mashed the button" then you could just increment a counter every time the player clicks the button to flap their wings. But not sure if you want to go that route